docs/getting_started/trivia_storage.md

```diff
@@ -25,7 +25,7 @@ storeLargePayload: state => {
 ## Per-user storage
 
 If you want to store things about your users and you want that information to be persisted across multiple conversations, we suggest you use [User Tags](../../advanced/tags).
-
+<!-- can't find in repo -->
```

Comment:
@dmytropanontko , I think we need to just remove link here.

> we suggest you use
we suggest you to use